services/user-management/src/main/java/com/foodie/user/AuthenticationController.java

```diff
@@ -0,0 +1,74 @@
+package com.foodie.user;
+
+import org.springframework.web.bind.annotation.RestController;
+
+import com.foodie.jwt.AuthenticationService;
+import com.foodie.models.contracts.AuthenticationRequest;
+import com.foodie.models.contracts.AuthenticationResponse;
+import com.foodie.models.contracts.RegisterRequest;
+import com.foodie.models.contracts.ResponseHandler;
+import com.foodie.models.users.User;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@RestController
+@RequestMapping("/api/v1/auth")
+@RequiredArgsConstructor
+public class AuthenticationController {
+
+    @Autowired
+    private final AuthenticationService service;
+
+    private Logger logger = LoggerFactory.getLogger(RoleController.class);
+
+    @PostMapping(path = "/register", produces = "application/json", consumes = "application/json")
+    public ResponseEntity<?> registerUser(@Valid @RequestBody RegisterRequest request) {
+
+        logger.info("AuthenticationController.registerUser() called with request: " + request);
```

Comment:
This should be `logger.info("AuthenticationController.registerUser() called with request: {request}");` and every other place where you use logger